.env.docker.example

```diff
@@ -14,7 +14,7 @@ SECURE_SSL_REDIRECT=False
 # Application Settings
 
 CONNECTED_APP_CLIENT_SECRET=...
-CONNECTED_APP_CALLBACK_URL=http://localhost:8080/callback
+CONNECTED_APP_CALLBACK_URL=http://localhost:8080/accounts/salesforce/login/callback/
```

Comment:
@davisagli It seems to work fine with just e.g. `http://localhost:8080/` (and staging has `https://metadeploy-dev.herokuapp.com/`) so I'll likely revert this.